examples/simpleapp/src/main/java/com/horizen/examples/SimpleAppModule.java

```diff
@@ -154,5 +156,9 @@ public void configureApp() {
                 .annotatedWith(Names.named("ForkConfiguration"))
                 .toInstance(forkConfigurator);
 
+        bind(Sc2ScConfigurator.class)
+                .annotatedWith(Names.named("Sc2ScConfiguration"))
+                .toInstance(new Sc2ScConfigurator(true, true, true) );
```

Comment:
Maybe can be better to move the parameter **canBuildProof** inside the node config since it is a local property of the node and not of the entire sidechain.

Reply:
will remove it to keep it simpler for the moment